src/main.rs

```diff
@@ -118,7 +119,8 @@ impl Group {
     pub async fn store_keypair(&self, protected_store: &veilid_core::ProtectedStore) -> Result<()> {
```

Comment:
```suggestion
    async fn store_keypair(&self, protected_store: &veilid_core::ProtectedStore) -> Result<()> {
```

Lets keep it private